### DISCUSSION TITLE
No recordings for a few species in the "Recordings" tab

### DISCUSSION THREAD

**Discussion by blogdemoi** [mcguirepr89/BirdNET-Pi#824] [Q&A] [answered]
Hi,

Today and yesterday I don't have any audio files to play in the "recordings" tab for a few species. If I take "common blackbird", the spectrograms are there but no media player appear. They did appear  in the "Today's Detection" tab though and played nicely.

In /home/arnauld/BirdSongs/Extracted/By_Date/2023-04-04/Merle_noir  (Merle noir, in French, is common blackbird).
I do have a lot of mp3s and pngs files

What could be wrong? As I didn't touch anything in my system.

Thank you.

Edit: I double-checked and the .mp3 in the screenshot below do exist. If I download it I can play it on my local computer.

**Comment by blogdemoi** [comment]
Here is a screenshot: 

![uzGV8Kc](https://user-images.githubusercontent.com/83965802/229956301-a6c2ce41-63e0-43fb-bb8a-5c930385202e.png)

**Comment by blogdemoi** [comment]
Same thing today, a few species don't have any play controls, just a spectrogram:

![DVr2SyB](https://user-images.githubusercontent.com/83965802/230226207-385244da-e17e-4527-8f46-dc986060876f.png)

However I just noticed that if I click on the spectrogram the page changes to this:

![qIosRrh](https://user-images.githubusercontent.com/83965802/230226316-4671f123-00db-4dac-910f-a6c64aa19334.png)

And then I can successfully play the recording. It's weird, as I have this only for 3/4 species, most of the species in the "Recordings" tab don't have this problem...

**Reply by ehpersonal38** [reply, marked as answer]
Heh, actually this isn't a bug at all. What's happening is I made a change to the code last month or to improve performance on the Recordings page. I noticed that the page would become virtually unusable when there was a large amount of media player elements visible on the screen at one time. So now, for pages on Recordings which have > 100 recordings, it just removes the media player altogether and reverts to a simpler "click-to-listen" type interface. That's why you only notice this for some species and not others. Hope that clears things up for you.

**Reply by blogdemoi** [reply]
Ah, ok, I understand. I was confused :-) Thanks for the clarification.
